src/cosmos/tx/v1beta1/tx.ts

```diff
@@ -1,37 +1,38 @@
 /* eslint-disable */
-import Long from "long";
-import _m0 from "protobufjs/minimal";
 import { Any } from "../../../google/protobuf/any";
-import { Coin } from "../../base/v1beta1/coin";
-import { CompactBitArray } from "../../crypto/multisig/v1beta1/multisig";
 import { SignMode, signModeFromJSON, signModeToJSON } from "../signing/v1beta1/signing";
-
+import { CompactBitArray } from "../../crypto/multisig/v1beta1/multisig";
+import { Coin } from "../../base/v1beta1/coin";
+import * as _m0 from "protobufjs/minimal";
+import { isSet, bytesFromBase64, base64FromBytes, DeepPartial, Exact, Long } from "../../../helpers";
 export const protobufPackage = "cosmos.tx.v1beta1";
-
 /** Tx is the standard type used for broadcasting transactions. */
+
 export interface Tx {
   /** body is the processable content of the transaction */
-  body?: TxBody;
+  body: TxBody;
```

Comment:
I imagine this is a good thing ;)

Reply:
- [ ] https://github.com/osmosis-labs/telescope/issues/235

Reply:
fixed. I realized that we have two perspectives and we can solve them separately: API optionality and proto optionality are different. I was thinking about API but as far as proto and TS are concerned, it makes sense to rollback all the type optionality to the proto defaults.